website_lazy_load_image/models/ir_ui_view.py

```diff
@@ -32,5 +32,6 @@ def render_template(self, template, values=None, engine='ir.qweb'):
                 src = img.attrib['src']
                 img.attrib['src'] = self.LAZYLOAD_DEFAULT_SRC
                 img.attrib['data-src'] = src
-            res = lxml.etree.tostring(html, method='html', encoding='UTF-8')
+            res = lxml.etree.tostring(html, method='html', encoding='UTF-8', 
```

Comment:
seems to make sense but according to lxml docs / issues this should work OOTB since some versions.
Which lxml version are you using?
@tarteo do you experience the same issue?

Reply:
Yes this seems to fix it.